cpuinfo_test.go

```diff
@@ -402,3 +466,16 @@ func TestCPUInfoParseRISCV64(t *testing.T) {
 		t.Errorf("want ModelName %v, have %v", want, have)
 	}
 }
+
+func TestCPUInfoParseLOONG64(t *testing.T) {
```

Comment:
```suggestion
func TestCPUInfoParseLoong64(t *testing.T) {
```

Reply:
Other architectures are capitalized. I think it may be better to keep consistent

Reply:
No. `ARM` `PPC` and `RISCV` are all acronyms ("Advanced RISC Machine", "PowerPC" for the first two, while "RISC-V" obviously cannot be abbreviated further except into "RV"), but Loong is a normal word. According to [common Go coding style](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), it's better to use `Loong`. (In this file the MIPS case is spelled `Mips`, which technically should become `MIPS` instead because the MIPS name is also an acronym, but you don't need to change it this time to keep the PR focused.)

Reply:
thank you, I have changed the function name